vnext/Microsoft.ReactNative/Fabric/Composition/Modal/WindowsModalHostViewComponentView.h

```diff
@@ -57,7 +62,11 @@ struct WindowsModalHostComponentView
 
  private:
   HWND m_hwnd{nullptr};
-  winrt::Microsoft::ReactNative::ReactContext m_context;
+  uint64_t m_prevWindowID;
+  winrt::Microsoft::ReactNative::ReactContext m_reactContext;
```

Comment:
m_reactContext is already a member of the base ComponentView, so you dont need another copy here.
CompositionContext() is a member of the base Composition::ComponentView, so you dont need another copy here.